File: Python/Data Types/Strings/stringLength.py
# program to find length of a string using len() function

str = input("Enter a string: ")
len = len(str)
print(f"Length of string is : {len}")
    

#============================================================================
#============================================================================


# program to find length of a string without using len() function
    
str = input("Enter a string: ")
length = 0
for x in str:
    length += 1
print(f"Length of string is : {length}")

#============================================================================
#============================================================================

File: C++/String/stringLength.cpp
#include <iostream>
#include <string.h>
using namespace std;

// program to find string length using strlen() function

int main() {
    char string[20];
    int len;
    cout << "Enter a string: ";
    cin >> string;
    len =  strlen(string);
    cout << "Length of string is: " << len;
    return 0;
}


// ============================================================================
// ============================================================================


// program to find string length without using strlen() function

int main() {
    char string[20];
    int len = 0;
    cout << "Enter a string: ";
    cin >> string;
    for (int i=0; string[i] != '\0'; i++) {
        len++;
    }
    cout << "Length of string is: " << len;
    return 0;
}